secrets/spi/src/main/resources/application.conf

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+secretsProvider {
+  name = "vault"
+  name = ${?SECRETS_PROVIDER_NAME}
+
+  # Properties for the Vault provider.
+  vaultUri = ${?VAULT_URI}
+  vaultRoleId = ${?VAULT_ROLE_ID}
+  vaultSecretId = ${?VAULT_SECRET_ID}
+  vaultRootPath = ${?VAULT_ROOT_PATH}
+  vaultPrefix = ${?VAULT_PREFIX}
+  vaultNamespace = ${?VAULT_NAMESPACE}
```

Comment:
Is it possible to move the settings to the respective module, like `:secrets:vault` in this case? Then the SPI would not have to know about its implementations.

Reply:
Yes, I have added another commit which does exactly this. In theory, such a split would be possible for other abstractions, too; but since we are currently using worker-specific properties, the configuration can only be defined in the workers modules.